doc/nrf/ug_thingy91.rst

```diff
@@ -197,7 +197,7 @@ Building and programming using SEGGER Embedded Studio
 .. figure:: images/ses_thingy_configuration.png
```

Comment:
This screenshot must be updated.

Reply:
can we just "photoshop" it?

Reply:
added a new screenshot

Reply:
The new one has thingy91_nrf9160 as target, not thingy91_nrf9160ns - is that correct?

Reply:
No. I will fix

Reply:
new image with *ns is pushed now